packages/request-node/src/request/getLitCapacityDelegationAuthSig.ts

```diff
@@ -6,7 +6,8 @@ import { StatusCodes } from 'http-status-codes';
 import * as config from '../config';
 import { LitContracts } from '@lit-protocol/contracts-sdk';
 import { utils } from 'ethers';
-
+import { LIT_NETWORKS_KEYS } from '@lit-protocol/types';
```

Comment:
Is it too late to rename this to just `LIT_NETWORKS`?

```suggestion
import { LIT_NETWORKS } from '@lit-protocol/types';
```

Reply:
This is from lit SDK I cannot rename it